pom.xml

```diff
@@ -81,23 +81,81 @@
     </snapshotRepository>
   </distributionManagement>
 
+
+  <properties>
+
+    <!-- Maven general -->
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+    <!-- maven-compiler-plugin -->
+    <maven.compiler.source>1.6</maven.compiler.source>
+    <maven.compiler.target>1.6</maven.compiler.target>
```

Comment:
I think I asked about that already, but can't remember.
Given that java 7 is EOL - what keeps us on 6 here?